contrib/seeds/makeseeds.py

```diff
@@ -107,25 +113,26 @@ def parseline(line):
         'sortkey': sortkey,
     }
 
-def dedup(ips):
-    '''deduplicate by address,port'''
+def dedup(ips: List[Dict]) -> List[Dict]:
```

Comment:
Hrm I don't think the types are correct here. `ips` is a dict of (ip, port) tuples. Was this validated using mypy?

Reply:
Thanks for the mypy recommendation. I did some more tests (with mypy and good old print statements) and `ips` in `dedup` is indeed a list of dictionaries -- of the same format that `parseline()` produces as output. I do believe that the typing for `ips` is correct here.

However, testing with mypy did bring up a few other places where I forgot to indicate possible `None` return types. See the commit below the fixes it. 

Thank you for your time reviewing this :)

Reply:
Indeed, seems I was wrong about the input! But the return type of `dedup` is a list of tuples, rught?

Reply:
I don't believe so, each value in the list of returned values from `dedup` is one of the dictionaries from the list `ips`. Testing with mypy and print statements also seems to confirm that the returned format is the same as the input -- a list of dicts.

But if you think I am confused or have royally stuffed up my tests, please let me know so I can hopefully fix things.

Reply:
Oh, you're right. It returns `d.values` which are the values, not the items or keys of the dict.